firmware/hw_layer/drivers/can/can_msg_tx.cpp

```diff
@@ -59,9 +59,25 @@ CanTxMessage::CanTxMessage(CanCategory p_category, uint32_t eid, uint8_t dlc, si
 #endif // HAS_CAN_FRAME
 }
 
-CanTxMessage::~CanTxMessage() {
-#if EFI_SIMULATOR
+CanTxMessage::~CanTxMessage(){
```

Comment:
i like that space

considering how amazing this PR is I am fine loosing the space

Reply:
fixed, it was vscode's auto-formatting, do we have any specific tool to format the files?